frontend/src/slices/auth/actions.ts

```diff
@@ -2,6 +2,8 @@ import { createAsyncThunk } from '@reduxjs/toolkit';
 
 import { type AsyncThunkConfig } from '~/libs/types/types.js';
 import {
+  type UserAuthResponseDto,
+  type UserSignInRequestDto,
   type UserSignUpRequestDto,
   type UserSignUpResponseDto,
 } from '~/packages/users/users.js';
```

Comment:
```suggestion
import type {
  UserAuthResponseDto,
  UserSignInRequestDto,
  UserSignUpRequestDto,
  UserSignUpResponseDto,
} from '~/packages/users/users.js';
```

Reply:
Linter said that I should prefer version with type near each import.
Avoid import/export type { Type } from './module'. Prefer import/export { type Type } from './module'.eslint[no-restricted-syntax](https://eslint.org/docs/latest/rules/no-restricted-syntax)